### ISSUE TITLE
New aggregate badge for organizations

### ISSUE THREAD

**Issue by giorrrgio** [PUGX/badge-poser#96]
It'd be cool to add a badge for organizations showing the total downloads of all the mantained packages.

**Comment by stof** [comment]
I don't think Packagist exposes such data though

**Comment by giorrrgio** [comment]
@stof you are right, but we can obtain the information with some effort (Idea: check which packages of the github organization have composer.json files and aggregate their packagist data)

**Comment by JellyBellyDev** [comment]
a little too `curl` to show a small badge don't you think? :D
